.travis.yml

```diff
@@ -141,6 +144,7 @@ requires:
     - libglade2-dev
     - libgsf-1-dev
```

Comment:
```suggestion
```

Reply:
Good call. I've removed all the gsf's from the build deps. New test up.